src/runtime-tools/win64/onefuzz.ps1

```diff
@@ -6,6 +6,8 @@ $env:ONEFUZZ_ROOT = "C:\onefuzz"
 $env:ONEFUZZ_TOOLS = "C:\onefuzz\tools"
 $env:LLVM_SYMBOLIZER_PATH = "llvm-symbolizer"
 $env:RUST_LOG = "info"
+$env:DOTNET_ROOT = "c:\onefuzz\tools\dotnet"
+[Environment]::SetEnvironmentVariable("DOTNET_ROOT", "c:\onefuzz\tools\dotnet", "Machine")
```

Comment:
I'm guessing it wasn't immediately obviously that it was necessary to set this at machine scope and also set it in the current session. Maybe worth a comment?

Also, consider reusing the session's env var when setting the machine-scoped one, just to emphasize that they're identical. I.e.
```suggestion
[Environment]::SetEnvironmentVariable("DOTNET_ROOT", $env:DOTNET_ROOT, "Machine")
```